### PR TITLE
Add note for WebDAV and TLSv1.1 / TLSv1.2

### LINKED ISSUE #34: dav/clients/windows additions for HTTPS/SSL
Hi,

quoting from an older ownCloud forums thread (https://forum.owncloud.org/viewtopic.php?f=23&t=11246&p=61241#p61202) which maybe could also be added to http://sabre.io/dav/clients/windows/

> Worth noting for anyone who's still having this problem after making the registry change to enable Basic authentication is that the WebDAV implementation in Windows 7 and 8 requires a "poor" configuration of SSL on the server side - the WinHTTP API can only use an SSLv2 Hello message, and will only negotiate up to TLSv1.0. So if you've been following current security trends and restricted your HTTPS server to TLSv1.1 and above (which is a "good thing" as a rule) then the Windows WebDAV client will give you issues including the 1244 Error.

### PR DESCRIPTION
**Pull request by RealRancor**
closes #34 

Source of this infos are:

https://msdn.microsoft.com/en-us/library/windows/desktop/aa382925%28v=vs.85%29.aspx
https://forum.owncloud.org/viewtopic.php?f=23&t=11246&p=61241#p61202

### CODE REVIEW

File: source/dav/clients/windows.md [hunk 1]
```diff
@@ -278,6 +278,10 @@ one HTTPS-based domain off one ip address, or make sure that it's the default.
 In the case of apache, the default https server would be the top-most
 virtualhost definition.
 
+Furthermore it was reported that the WinHTTP API (used in the Windows WebDAV client)
+doesn't support TSLv1.1 / TSLv1.2 connections. If you have restricted your server
```
Comment (Hywan) [inline source/dav/clients/windows.md]: Please, remove spaces around `/`.

File: source/dav/clients/windows.md [hunk 2]
```diff
@@ -278,6 +278,10 @@ one HTTPS-based domain off one ip address, or make sure that it's the default.
 In the case of apache, the default https server would be the top-most
 virtualhost definition.
 
+Furthermore it was reported that the WinHTTP API (used in the Windows WebDAV client)
```
Comment (Hywan) [inline source/dav/clients/windows.md]: Add a link to WinHTTP API please (the one on the description of this PR).

**Comment by RealRancor** [comment]
@Hywan 
Done, thanks for the suggestions.

**Comment by evert** [comment]
Is this a confirmed issue in Windows 7/8?

**Comment by RealRancor** [comment]
Confirmed here:

https://forum.owncloud.org/viewtopic.php?f=23&t=11246&p=61241#p61202

**Comment by evert** [comment]
Sounds good to me then =)

**Pull request merged**